### PR TITLE
chore: Cleanup Environments

### PR DESCRIPTION
**Pull request by doitian**
* Make the bootnode location names consistent in the config file
* Remove obsoleted staging environment. The bootnode listed as staging has joined the testnet Aggron.

### CODE REVIEW

File: resource/ckb.toml [hunk 1]
```diff
@@ -56,9 +53,8 @@ listen_addresses = ["/ip4/0.0.0.0/tcp/8115"] # {{
...
...XJJaKzQEo3rb",\n  # Beijing, Asia\n  "/ip4/39.107.100.85/tcp/8114/p2p/QmeQwD2GGuZyFzDPbQEKFJUjmNaY9FG3X7WHPL823zFD3a",\n  # Shanghai, Asia\n  "/ip4/47.103.44.208/tcp/8114/p2p/QmWVt9kNFv8XM1CMqeJj2nmG4tJ4ViqgwzJMr7yYynp6qd",\n  # Shenzhen, Asia\n  "/ip4/120.24.85.239/tcp/8114/p2p/QmaJP1sDiWZuwAEMghNHt7TrTgaMCyYaEMMYLc4YvoUGSV",\n  # Hong Kong, Asia\n  "/ip4/47.91.238.128/tcp/8114/p2p/QmTjLAewCM6SivjpW7BJfSj1ABuPA7x6FsFu5ga7Xy2xig",\n  # Jakarta, Asia\n  "/ip4/147.139.137.53/tcp/8114/p2p/QmQBLw9TqkS8yu2Kg8UtiYzvxEQ7DfKiXLx8iD7bF8XRyj",\n  # California, North America\n  "/ip4/13.52.18.181/tcp/8114/p2p/QmcEK1wUR287qSYdw8eHNWeQrFitQsCaZHHTM9wgvakxnS",\n  # Singapore, Asia\n  "/ip4/18.140.240.153/tcp/8114/p2p/QmVi7reKhqVnoBYzW2nJYnrujVeckrZXhwuYbX7P2whPJg",\n  # Quebec, North America\n  "/ip4/35.183.172.68/tcp/8114/p2p/QmNRAvtC6L85hwp6vWnqaKonJw3dz1q39B4nXVQErzC4Hx",\n  # London, Europe\n  "/ip4/3.10.216.39/tcp/8114/p2p/QmagxSv7GNwKXQE7mi1iDjFHghjUpbqjBgqSot7PmMJqHA",\n  # Sydney, Oceania\n  "/ip4/3.105.209.193/tcp/8114/p2p/QmQidJaxciY3NT2PjsaCR4Gz8vB8kFn3Avwz96u6b3jGc1",\n  # Sao Paulo, South America\n  "/ip4/18.229.214.173/tcp/8114/p2p/QmVeeCh81GTLGRwB7vRHXeTRdUHRYcfn6qKEfewhtiRJZC"\n]
```
Comment (quake) [inline resource/ckb.toml]: Tokio => Tokyo :)

**Comment by doitian** [comment]
Pushed to fix the Tokyo typo. Please review again.

File: resource/ckb.toml [hunk 2]
```diff
@@ -56,9 +53,8 @@ listen_addresses = ["/ip4/0.0.0.0/tcp/8115"] # {{
...
```
Comment (yangby-cryptape) [inline resource/ckb.toml]: ### Typo

- `Amarica`: `Virginia, North Amarica` -> `Virginia, North America`

Comment (doitian) [reply resource/ckb.toml]: Fixed

**Comment by doitian** [comment]
bors r=zhangsoledad,driftluo,yangby-cryptape

**Pull request merged**